.github/workflows/wpcloud.yml

```
@@ -122,25 +121,39 @@ jobs:
           echo "  User wpcom-jetpackisbestpack-default-237778992" >> ~/.ssh/config
           echo "  IdentityFile ~/.ssh/id_site" >> ~/.ssh/config
           echo "  IdentitiesOnly yes" >> ~/.ssh/config
-
           echo "::endgroup::"
 
           echo "::group::Transferring wpcomsh to the testing server"
-          ssh jpwpcomsh "wp --skip-plugins --skip-themes dereferenced freshen > /dev/null 2>&1" || CODE=$?
-          echo "wp dereferenced freshen has exited with code $CODE"
+          # This can give errors if the previous state was broken, so ignore them
+          ssh jpwpcomsh "wp --skip-plugins --skip-themes dereferenced freshen > /dev/null 2>&1" || echo "wp dereferenced freshen has exited with code $?"
           ssh jpwpcomsh "rm -rf /tmp/old-* > /dev/null 2>&1"
-          pnpm jetpack rsync --non-interactive wpcomsh jpwpcomsh:~/htdocs/wp-content/mu-plugins
+          pnpm jetpack rsync --non-interactive wpcomsh jpwpcomsh:/srv/htdocs/wp-content/mu-plugins
           scp -r projects/plugins/wpcomsh/bin jpwpcomsh:/srv/htdocs/wp-content/mu-plugins/wpcomsh/
           scp -r projects/plugins/wpcomsh/tests jpwpcomsh:/srv/htdocs/wp-content/mu-plugins/wpcomsh/
-          # The test on WP Cloud won't use `phpunit-select-config`, so select config manually.
-          scp projects/plugins/wpcomsh/phpunit.9.xml.dist jpwpcomsh:/srv/htdocs/wp-content/mu-plugins/wpcomsh/phpunit.xml.dist
-
           echo "::endgroup::"
 
-          echo "::group::execution"
-          ssh jpwpcomsh "/srv/htdocs/wp-content/mu-plugins/wpcomsh/bin/run-phpunit-tests.sh" || CODE=$?
+          # Do a basic check to verify the site is loading
+          echo "::group::Verify things load"
+          SITE_URL=$(ssh jpwpcomsh "wp option get siteurl" 2>/dev/null)
+          CODE=$?
+          if [[ $CODE -ne 0 ]]; then
+            echo 'Unable to run a basic `wp` command! Something is wrong with the site.'
+          else
```

Let's throw in some minimal validation of the response before trying to curl it
```suggestion
          elif [[ ! "$SITE_URL" =~ ^https?://[a-z0-9_.]+$ ]]; then
            echo "Site url doesn't seem to be a URL"
            CODE=1
          else
```
Adjust the regex there as necessary. POSIX extended regex syntax (i.e. same as `grep -E` or `sed -E`).

---

35211f1137